webpack/reducers/forms_reducer.js

```diff
@@ -1,13 +1,20 @@
 import {
+  FETCH_FORM_FULFILLED,
   FETCH_FORMS_FULFILLED
 } from '../actions/types';
 import _ from 'lodash';
 
-const initialState = {'loading': true};
+const initialState = null;
```

Comment:
It's generally easier to work with things if you set the initial state to an empty hash. That way, can always call _.values(state.forms). If you do that before forms are fetched, it's going to be sad.